styleguide/_themes/derek/scss/components/ticker.scss

```diff
@@ -0,0 +1,88 @@
+@import '../../../global/scss/swatch';
+@import '../../../global/scss/fonts';
+@import '../../../global/scss/bootstrap_variables';
+@import '../../../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/clearfix';
+@import '../../../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/grid';
```

Comment:
If we're gonna have an externally accessible css file, I'm not sure placing it inside the derek components directory makes sense.  Should this be in a `/styleguide/css/ticker.scss` file instead and then just have derek.scss include that instead?   (I'm not sure that makes sense or not, but having this inside derek specifically seems a little odd to me).